### ISSUE TITLE
Prevent redirects when generating pages

### ISSUE THREAD

**Issue by dysbulic** [MetaFam/TheGame#424]
Intermittently when the site is being generated it is erroring out when a redirect is returned:

```
15:21:47.372 | @metafam/web: Unhandled error during request: Error: `redirect` can not be returned from getStaticProps during prerendering (/player/sparrow)
-- | --
15:21:47.372 | @metafam/web: See more info here: https://err.sh/next.js/gsp-redirect-during-prerender
15:21:47.372 | @metafam/web:     at renderToHTML (/vercel/workpath0/packages/web/node_modules/next/dist/next-server/server/render.js:30:1174)
15:21:47.372 | @metafam/web:     at processTicksAndRejections (internal/process/task_queues.js:97:5)
15:21:47.372 | @metafam/web:     at async renderReqToHTML (/vercel/workpath0/packages/web/.next/serverless/pages/player/[username].js:8289:22)
15:21:47.372 | @metafam/web:     at async Object.exportPage [as default] (/vercel/workpath0/packages/web/node_modules/next/dist/export/worker.js:12:92)
```
My assumption is it's [this section](https://github.com/MetaFam/TheGame/blob/master/packages/web/pages/player/%5Busername%5D.tsx#L183-L191) which I don't completely understand. It seems like it is being called from the list of usernames, so `username` should be set.

The other option is [here](https://github.com/MetaFam/TheGame/blob/master/packages/web/pages/player/%5Busername%5D.tsx#L193-L195). It is getting called with a lowercase name, so that isn't it, but maybe the first call is failing and the second is succeeding.

Adding retries (#419) might fix it if it is that it is coming from a backend failure.

**Comment by pakokrew** [comment]
https://github.com/MetaFam/TheGame/issues/456
`desination`
fixed in #439

**Comment by dan13ram** [comment]
Redirects are returned at build time only because hasura times out.
Closing this in favor of #508
